src/config/config.example.ts

```diff
@@ -3,7 +3,7 @@ export const config = {
   host: 'localhost',
   port: 8889,
   username: 'root',
-  password: 'root',
+  password: '',
   database: 'megak_bonus',
   logging: true,
   synchronize: true,
```

Comment:
Not sure if i should include the file. Some of us work on macOS and some on windows so we can have different configs.

Reply:
Można wszystko wrzucić w komentarz. Można nawet zrobić 2 konfiguracje pod Windows i macOS/linux. Kto będzie chciał jaki config to sobie odkomentuje.